CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing Guide
```

Comment:
Did you see [docs/contributing.md](https://github.com/mariusvniekerk/condax/blob/main/docs/contributing.md)?

Reply:
Oh lol no XD

Reply:
For a Conda-centric tool like condax, I wonder how justified it is to compel contributors to use the Hatch CLI for development. Thanks to build-backend standardization, using Hatch is not necessary...

An alternative would be:

```
conda create --name condax-dev pip
conda activate condax-dev
pip install -e .
```

Here it would make sense to have a separate `requirements-dev.txt` so that one could do

```
pip install -r requirements-dev.txt
```

I'm wondering if it's possible to bring such a `requirements-dev.txt` back into the `pyproject.toml` via something like [hatch-requirements-txt](https://github.com/repo-helper/hatch-requirements-txt)...

Reply:
>But having a list of dependencies in more than one place is messy
:+1:

> which is why I don't like having a requirements.txt if the dependencies are also going to be in pyproject.toml

They don't have to go into the pyproject.toml. See [hatch-requirements-txt](https://github.com/repo-helper/hatch-requirements-txt).

Reply:
Using `requirements-dev.txt` as an optional dependency group like what I proposed above has been feature-requested in https://github.com/repo-helper/hatch-requirements-txt/issues/6